Sources/General/ZLThumbnailViewController.swift

```diff
@@ -296,7 +308,15 @@ class ZLThumbnailViewController: UIViewController {
         collectionView.frame = CGRect(x: insets.left, y: 0, width: totalWidth, height: view.frame.height)
         collectionView.contentInset = UIEdgeInsets(top: collectionViewInsetTop, left: 0, bottom: bottomViewH, right: 0)
         collectionView.scrollIndicatorInsets = UIEdgeInsets(top: insets.top, left: 0, bottom: bottomViewH, right: 0)
-        
+
+        let scrollToBottomSize = 48.0
+        let scrollToBottomX = totalWidth - scrollToBottomSize - 15.0
+        let scrollToBottomY = view.frame.height - insets.bottom - bottomViewH - scrollToBottomSize - 5.0
+        scrollToBottomBtn.frame = CGRect(origin: CGPoint(x: scrollToBottomX,
```

Comment:
Code format: 
```swift
scrollToBottomBtn.frame = CGRect(
    origin: CGPoint(x:, y:),
    size: CGSize(width:, height:)
)
```

Reply:
I am not sure If I got your right, updated, but that was basically 2 new lines removal